__TESTS__/text-diff.spec.tsx

```diff
@@ -27,4 +26,40 @@ describe('TextDiff', () => {
 
     expect(output.innerHTML).toBe(testOutput);
   });
+
+  it('Show different characters ignoring case', async () => {
+    const randomCase = (str: string): string =>
+      str
+        .split('')
+        .map((character) =>
+          Math.round(Math.random()) === 0
+            ? character.toLowerCase()
+            : character.toUpperCase(),
+        )
+        .join();
+
+    userEvent.selectOptions(
+      // screen.getByLabelText(/diff-select-label/i),
+      screen.getByTestId('text-difference-options'),
+      ['Characters Ignore Case'],
+    );
+
+    const testInput1 = 'I am the Lorax';
+    const testInput2 = 'I speak for the trees';
+    const testOutput =
+      '<span style="color:grey">I </span><span style="color:green">spe</span><span style="color:grey">a</span><span style="color:red">m</span><span style="color:green">k</span><span style="color:grey"> </span><span style="color:green">for </span><span style="color:grey">the </span><span style="color:red">Lo</span><span style="color:green">t</span><span style="color:grey">r</span><span style="color:red">ax</span><span style="color:green">ees</span>';
+    const user = userEvent.setup();
+    render(<TextDiff />);
+
+    const input1 = screen.getByLabelText(/Text 1/i);
+    const input2 = screen.getByLabelText(/Text 2/i);
+    const output = screen.getByTestId('text-difference-output');
```

Comment:
Try this: `getByLabelText(/Choose Diff Options/i);`

Reply:
Tried this one but it doesn't work unfortunately.

Reply:
Weird, lets push the other changes without writing any new tests and I'll take a look at it to see if I can figure out the issue.

Reply:
Will do. I went through StackOverflow and it seems many people were having issues with testing MUI Select elements. I tried several ways mentioned in the posts but nothing has worked for me so far.